gulpfile.js

```diff
@@ -50,12 +50,12 @@ gulp.task('build.demo-js', function () {
 
 gulp.task('build.dist-css', function() {
     return gulp.src(src.css)
-        .pipe(plugins.if(/.scss/, plugins.sass({ style: 'compressed', noCache: true }))) // jshint ignore:line
+        .pipe(plugins.if(/.scss/, plugins.sass({ style: 'compressed', noCache: true })))
         .pipe(plugins.autoprefixer(autoprefixerBrowsers))
         .pipe(plugins.concat('baguetteBox.css'))
         .pipe(gulp.dest(dist.css))
         .pipe(plugins.concat('baguetteBox.min.css'))
-        .pipe(plugins.cssmin())
+        .pipe(plugins.cssmin({ compatibility: 'ie8' }))
```

Comment:
Should this be `ie7`?

Reply:
It's fine, we're officially supporting IE8+